### ISSUE TITLE
LLVM horizontal reduction intrinsics broken on AArch64

### ISSUE THREAD

**Issue by gnzlbg** [gnzlbg/packed_simd#15]
These include:

* floating-point `sum` and `product`
* integer `wrapping_sum` and `wrapping_product`
* horizontal min/max reductions: `min_element`/`max_element`
* bitwise horizontal reductions: `and`/`or`/`xor`
* mask reductions `all`,`any`, `none`

Reported to LLVM: https://bugs.llvm.org/show_bug.cgi?id=36796

**Comment by nikic** [comment]
This issue has been fixed upstream.

**Comment by gnzlbg** [comment]
@nikic has LLVM been updated to a version with the fix?

**Comment by nikic** [comment]
@gnzlbg Nope. The relevant monorepo commit is https://github.com/llvm/llvm-project/commit/aa7cfa75f97ed2b7f27b22f626353bc29d03819b#diff-4cd9888950373f65c4898b144777d044. Right now we're still on the LLVM 8 branch (unsure if that means that we will no longer track LLVM trunk?)

**Comment by SparrowLii** [comment]
@nikic It looks like it has been fixed? As it passed the tests normally on my aarch64 machine.

**Comment by nikic** [comment]
@SparrowLii Yes, the relevant LLVM upgrade has happened a long time ago by now :)
